lib/install/initializers.rb

```diff
@@ -0,0 +1,10 @@
+sr_initializer_path = Rails.root.join("config/initializers/stimulus_reflex.rb")
+cr_initializer_path = Rails.root.join("config/initializers/cable_ready.rb")
+
+generate "stimulus_reflex:initializer" unless sr_initializer_path.exist?
+say "✅ StimulusReflex initializer created at config/initializers/stimulus_reflex.rb"
+
+generate "cable_ready:initializer" unless cr_initializer_path.exist?
+say "✅ CableReady initializer created at config/initializers/cable_ready.rb"
```

Comment:
This should live in the CableReady installer

Reply:
As I said earlier, it will live in the CableReady installer *also*, but the goal is for people to not run a CR installer if they are using SR. Slightly more work for us, but way less complicated for our developers.

Totally worth it.